SPECS/tdnf/tdnf.spec

```
@@ -72,9 +78,19 @@ Group:          Development/Libraries
 %description cli-libs
 Library providing cli libs for tdnf like clients.
 
+%package plugin-metalink
+Summary:        tdnf plugin providing metalink functionality for repo configurations
+Group:          Applications/RPM
+Requires:       tdnf = %{version}-%{release}
```

Nit-pick: for requirements within the same spec, we try to use the macros. Just a consistency thing.
```suggestion
Requires:       %{name} = %{version}-%{release}
```